roofit/hs3/test/histfactory.py

```diff
@@ -0,0 +1,178 @@
+import unittest
+import ROOT
+
+
```

Comment:
```suggestion

msg = ROOT.RooMsgService.instance()
msg.setGlobalKillBelow(ROOT.RooFit.WARNING)

```
Make the test less verbose